esp-hal/MIGRATING-0.21.md

```diff
@@ -176,3 +177,18 @@ In case of any error you should drop the transfer and restart it.
 +            },
 +        };
 ```
+
+## Removed `peripheral_input` and `into_peripheral_output` from GPIO pin types
+
+Creating peripheral interconnect signals now consume the GPIO pin used for the connection.
+
+The previous signal function have been replaced by `split`. This change affects the following APIs:
```

Comment:
Is it worth mentioning that the `peripheral_input` and `into_peripheral_output` now exist on drivers instead of the raw pin itself? This entry sounds like it's been removed completely.

Reply:
This is the point where some technical writing background could come in handy. I was hoping mentioning the pin types makes this clear, but apparently not.